docker/config/Dockerfile

```diff
@@ -9,4 +9,13 @@ RUN apt-get install -y $(apt-cache depends contrail-config | egrep -v "superviso
 RUN dpkg --unpack contrail-config*.deb
 RUN rm -f /var/lib/dpkg/info/contrail-config.postinst
 
+# TMP Apply patch from review-id https://review.opencontrail.org/#/c/12863/1/src/config/api-server/utils.py
+RUN curl -s https://review.opencontrail.org/cat/12863%2C1%2Csrc/config/api-server/utils.py%5E1 | gunzip > /tmp/utils.py.orig
+RUN curl -s https://review.opencontrail.org/cat/12863%2C1%2Csrc/config/api-server/utils.py%5E0 | gunzip >/tmp/utils.py
+RUN diff -up /tmp/utils.py.orig /tmp/utils.py > /tmp/utils.py.diff || true
+RUN patch /usr/lib/python2.7/dist-packages/vnc_cfg_api_server/utils.py /tmp/utils.py.diff
```

Comment:
On Aug 5, 2015, at 7:22 PM, Ananth Suryanarayana <notifications@github.com<mailto:notifications@github.com>> wrote:


In docker/config/Dockerfile<https://github.com/pedro-r-marques/examples/pull/1#discussion_r36376531>:

> @@ -9,4 +9,13 @@ RUN apt-get install -y $(apt-cache depends contrail-config | egrep -v "superviso
>  RUN dpkg --unpack contrail-config*.deb
>  RUN rm -f /var/lib/dpkg/info/contrail-config.postinst
>
> +# TMP Apply patch from review-id https://review.opencontrail.org/#/c/12863/1/src/config/api-server/utils.py
> +RUN curl -s https://review.opencontrail.org/cat/12863%2C1%2Csrc/config/api-server/utils.py%5E1 | gunzip > /tmp/utils.py.orig
> +RUN curl -s https://review.opencontrail.org/cat/12863%2C1%2Csrc/config/api-server/utils.py%5E0 | gunzip >/tmp/utils.py
> +RUN diff -up /tmp/utils.py.orig /tmp/utils.py > /tmp/utils.py.diff || true
> +RUN patch /usr/lib/python2.7/dist-packages/vnc_cfg_api_server/utils.py /tmp/utils.py.diff


But we have to clone the entire git repo for that right.. I was trying to avoid that.

My understanding is that the command above downloads the patch file.

…<x-msg://14/#>
On Wed, Aug 5, 2015 at 7:04 PM, Pedro Marques <notifications@github.com<mailto:notifications@github.com>> wrote: In docker/config/Dockerfile <https://github.com/pedro-r-marques/examples/pull/1#discussion_r36375870>: > @@ -9,4 +9,13 @@ RUN apt-get install -y $(apt-cache depends contrail-config | egrep -v "superviso > RUN dpkg --unpack contrail-config*.deb > RUN rm -f /var/lib/dpkg/info/contrail-config.postinst > > +# TMP Apply patch from review-id https://review.opencontrail.org/#/c/12863/1/src/config/api-server/utils.py > +RUN curl -s https://review.opencontrail.org/cat/12863%2C1%2Csrc/config/api-server/utils.py%5E1 | gunzip > /tmp/utils.py.orig > +RUN curl -s https://review.opencontrail.org/cat/12863%2C1%2Csrc/config/api-server/utils.py%5E0 | gunzip >/tmp/utils.py > +RUN diff -up /tmp/utils.py.orig /tmp/utils.py > /tmp/utils.py.diff || true > +RUN patch /usr/lib/python2.7/dist-packages/vnc_cfg_api_server/utils.py /tmp/utils.py.diff If you look in the review, in DOWNLOAD its has the following: git fetch https://review.opencontrail.org/Juniper/contrail-controller refs/changes/63/12863/1 && git format-patch -1 --stdout FETCH_HEAD That is simpler than what you have. — Reply to this email directly or view it on GitHub <https://github.com/pedro-r-marques/examples/pull/1/files#r36375870>.

—
Reply to this email directly or view it on GitHub<https://github.com/pedro-r-marques/examples/pull/1/files#r36376531>.